src/vm/gdbjit.cpp

```diff
@@ -1730,22 +1730,24 @@ struct Elf_Symbol {
     Elf_Symbol() : m_name(nullptr), m_off(0), m_value(0), m_section(0), m_size(0) {}
 };
 
-static int countFuncs(const SymbolsInfo *lines, int nlines)
+template <class T, class U>
+static int countFuncs(NewArrayHolder<T> &arr, int n, U T::*ilOffset)
```

Comment:
This is unnecessarily complex. You don't need to pass in the ilOffset member pointer. The following works just fine (provided you also change the callers to not to pass the iloffset argument). Could you please change it that way?
```C++
template <class T>
static int countFuncs(T &arr, int n)
{
    int count = 0;
    for (int i = 0; i < n; i++) {
        if (arr[i].ilOffset == ICorDebugInfo::PROLOG)
        {
            count++;
        }
    }
    return count;
}

template <class T>
static int getNextPrologueIndex(int from, T &arr, int n)
{
    for (int i = from; i < n; ++i) {
        if (arr[i].ilOffset == ICorDebugInfo::PROLOG)
        {
            return i;
        }
    }
    return -1;
}

```

Reply:
Thank you. I've removed unnecessary parameter from this functions.